modules/javafx.controls/src/test/java/test/javafx/scene/control/MultipleSelectionModelImplTest.java

```diff
@@ -1396,4 +1396,53 @@ public void test_jdk_8088896() {
         assertEquals(2, model.getSelectedItems().size());
         assertEquals(1, counter.get());
     }
+
+    // Test for MultipleSelectionModelBase.SelectedIndicesList#set(int index)
+    @Test public void testSelectedIndicesList_SetMethod() {
+        model.clearSelection();
+        model.setSelectionMode(SelectionMode.MULTIPLE);
+        model.select(1);
+
+        assertTrue(model.isSelected(1));
+    }
+
+    // Test for MultipleSelectionModelBase.SelectedIndicesList#set(int index, int end, boolean isSet)
+    @Test public void testSelectedIndicesList_SetRangeMethod() {
+        model.clearSelection();
+        model.setSelectionMode(SelectionMode.MULTIPLE);
+        model.selectAll();
+
+        assertEquals(data.size(), model.getSelectedItems().size());
+    }
+
+    // Test for MultipleSelectionModelBase.SelectedIndicesList#set(int index, int... indices)
+    @Test public void testSelectedIndicesList_SetIndicesMethod() {
+        model.clearSelection();
+        model.setSelectionMode(SelectionMode.MULTIPLE);
+        model.selectIndices(1, 2, 5);
+
+        assertTrue(model.isSelected(1));
+        assertTrue(model.isSelected(2));
+        assertTrue(model.isSelected(5));
+        assertFalse(model.isSelected(0));
+    }
+
+    // Test for MultipleSelectionModelBase.SelectedIndicesList#clear()
+    @Test public void testSelectedIndicesList_ClearMethod() {
+        model.setSelectionMode(SelectionMode.MULTIPLE);
+        model.selectIndices(1, 2, 5);
+        model.clearSelection();
+
+        assertTrue(model.getSelectedIndices().isEmpty());
```

Comment:
I recommend to also check the size.

Reply:
Isn't checking the size redundant in this case? `model.getSelectedIndices().isEmpty()` indicates that `model.getSelectedIndices().size()` is `0`.